cypress.json

```diff
@@ -1,8 +1,12 @@
 {
-  "baseUrl": "https://folio-snapshot.dev.folio.org",
+  "baseUrl": "http://localhost:3000/",
```

Comment:
Do we need these changes here?

Reply:
no. And may be it's good to add config file example into md file and add such files into .gitignore. Need to discuss with the all Team.

Reply:
I will revert this change at the end of review, before merge in master

Reply:
done